pkg/trigger.go

```diff
@@ -66,7 +66,12 @@ func NewCmdTriggerBackup(clientGetter genericclioptions.RESTClientGetter) *cobra
 			}
 
 			for _, session := range backupConfig.Spec.Sessions {
-				_, err := triggerBackup(backupConfig, session)
+				if len(sessions) != 0 &&
```

Comment:
Is it needed to check `len(sessions) != 0` ?
I think `!slices.Contains(sessions, session.Name)` automatically handle empty slice.

Reply:
It is needed when no session is provided using the `sessions` flag and that means we need to trigger backup for all sessions provided in `BackupConfiguration`. If we don't check `len(sessions) != 0`, `slices.Contains()` will return `false` as there is no such element that matches the target element.